{"uuid":"fa61d710-2ff0-4d17-9654-648a5aeb4f02","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["e4a90213-233f-4472-a8cd-57e5993164bc"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696714847702,"stop":1696714847702},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696714847792,"stop":1696714847792},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696714847839,"stop":1696714847840},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696714849850,"stop":1696714849850}],"start":1696714847575,"stop":1696714849851}